apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/dokarkiv/DokarkivClient.java

```diff
@@ -58,33 +59,32 @@ public void gjenopprett(RsDollyUtvidetBestilling bestilling, DollyPerson dollyPe
                 dokarkivRequest.getAvsenderMottaker().setNavn(String.format("%s, %s%s", avsender.getFornavn(), avsender.getEtternavn(), isNull(avsender.getMellomnavn()) ? "" : ", " + avsender.getMellomnavn()));
             }
 
-            bestilling.getEnvironments().forEach(environment -> {
+            bestilling.getEnvironments().stream()
+                    .filter(StringUtils::isNotBlank)
+                    .forEach(environment -> {
 
                 if (!transaksjonMappingService.existAlready(DOKARKIV, dollyPerson.getHovedperson(), environment) || isOpprettEndre) {
-                    try {
-                        DokarkivResponse response = dokarkivConsumer.postDokarkiv(environment, dokarkivRequest).block();
-                        if (nonNull(response)) {
-                            status.append(isNotBlank(status) ? ',' : "")
-                                    .append(environment)
-                                    .append(":OK");
 
-                            saveTransaksjonId(response, dollyPerson.getHovedperson(),
-                                    progress.getBestilling().getId(), environment);
-                        }
+                    var response = dokarkivConsumer.postDokarkiv(environment, dokarkivRequest).block();
+                    if (nonNull(response) && isBlank(response.getFeilmelding())) {
+                        status.append(',')
+                                .append(environment)
+                                .append(":OK");
 
-                    } catch (RuntimeException e) {
+                        saveTransaksjonId(response, dollyPerson.getHovedperson(),
+                                progress.getBestilling().getId(), environment);
+                    } else {
 
-                        status.append(isNotBlank(status) ? ',' : "")
+                        status.append(',')
                                 .append(environment)
-                                .append(':')
-                                .append(errorStatusDecoder.decodeRuntimeException(e));
-
-                        log.error("Feilet å legge inn person: {} til Dokarkiv miljø: {}",
-                                dokarkivRequest.getBruker().getId(), environment, e);
+                                .append(":FEIL=Teknisk feil se logg! ")
+                                .append(nonNull(response) ?
+                                        errorStatusDecoder.encodeStatus(response.getFeilmelding()) :
+                                        "UKJENT");
                     }
                 }
             });
-            progress.setDokarkivStatus(status.toString());
+            progress.setDokarkivStatus(isNotBlank(status) ? status.substring(1) : null);
```

Comment:
StringUtils har substring(), den kanskje kan være 
`progress.setDokarkivStatus(StringUtils.substring(status, 1));`
men substring sjekker bare for null, ikke isNotBlank() hvis det er viktig

Reply:
Flott, sjekker ut det.